### DISCUSSION TITLE
Error. Was working fine until I put 3 monitors instead of 2.

### DISCUSSION THREAD

**Discussion by 2Kbummer** [Crozzers/screen_brightness_control#32] [Q&A]
[I am not sure how to link my code directly. Its in my github page.](https://github.com/2Kbummer/Screen-Dimmer/blob/main/screen_dimmer.py)
Traceback (most recent call last):
  File "screen_dimmer.py", line 126, in <module>
  File "screen_dimmer.py", line 30, in main
  File "screen_dimmer.py", line 116, in key_lists
  File "screen_brightness_control\__init__.py", line 44, in get_brightness
  File "screen_brightness_control\__init__.py", line 776, in __brightness
screen_brightness_control.helpers.ScreenBrightnessError: 
        no valid output was received from brightness methods

looks like it takes longer than 3 seconds to run sbc.get_brightness(display=monitor) for this older monitor from 2007. The other 2 monitors are identical.
![image](https://github.com/Crozzers/screen_brightness_control/assets/68630494/7c261396-d783-40b5-98c2-5aa3d07811a0)

**Comment by Crozzers** [comment]
Hi there!

Have you checked that your third monitor supports MCCS (monitor control command set) or DDC/CI (display data channel)? You may need to go into your monitor's on screen menu and enable it.

As for the delay, I'm hoping the above suggestion would fix it, otherwise I don't think there's much that I can do. This library is very dependent on how well the monitor manufacturer has implemented the DDC spec, which isn't always done well.
